src/controllers/controllerengine.cpp

```diff
@@ -835,13 +835,25 @@ void ControllerEngineConnectionScriptValue::disconnect() {
    Input:   the ControllerEngineConnection to trigger
    -------- ------------------------------------------------------ */
 void ControllerEngine::triggerControl(const ControllerEngineConnection conn) {
-    ControlObjectScript* coScript = getControlObjectScript(conn.key.group, conn.key.item);
+    if (m_pEngine == nullptr) {
+        return;
+    }
 
-    if (m_pEngine == nullptr || coScript == nullptr) {
+    ControlObjectScript* coScript = getControlObjectScript(conn.key.group, conn.key.item);
+    if (coScript == nullptr) {
         return;
     }
 
-    coScript->emitValueChanged();
+    QScriptValueList args;
```

Comment:
Yea it could just as easily be a struct. 

It should be a tiny change -- I don't want to leave an opportunity for someone to change one place but not the other.